### PR TITLE
Carousel implementation with overflow-x

### PR DESCRIPTION
**Pull request by RyanL123**

### CODE REVIEW

**Comment by err53** [comment]
What the fuck is that PR name

File: src/components/Carousel.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import React from "react"
 import { Box } from "@chakra-ui/core"
 import Title from "./Title"
-
+import Slider from "./Slider"
 export default () => (
-    <Box marginTop="200px" px="10vw">
-        <Title sub="Lorem ipsum dolor" main="Lorem ipsum dolor sit amet" />
-        <Box height="500px">{/* carousel here */}</Box>
+    <Box marginTop="200px">
```
Comment (davidli3100) [inline src/components/Carousel.js]: test, and use appropriate breakpoints for different screen sizes

File: src/components/Carousel.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 import { Box } from "@chakra-ui/core"
 import Title from "./Title"
-
+import Slider from "./Slider"
 export default () => (
-    <Box marginTop="200px" px="10vw">
-        <Title sub="Lorem ipsum dolor" main="Lorem ipsum dolor sit amet" />
-        <Box height="500px">{/* carousel here */}</Box>
+    <Box marginTop="200px">
+        <Box px="10vw">
```
Comment (davidli3100) [inline src/components/Carousel.js]: same as above, vw is bad practice for responsiveness

File: src/components/Footer.js [hunk 3]
```diff
@@ -1,44 +1,63 @@
...
-                    fluid(maxWidth: 2000, maxHeight: 700) {
-                        ...GatsbyImageSharpFluid_withWebp
-                    }
-                }
-            }
-        }
-    `)
-    return (
+export default () => (
+    <Box px="10vw" py="75px" color="blue.500">
```
Comment (davidli3100) [inline src/components/Footer.js]: don't use vw, it's bad practice for responsiveness, just use charka default breakpints

File: src/components/Footer.js [hunk 4]
```diff
@@ -1,44 +1,63 @@
...
         <Box
-            textAlign="center"
             display="flex"
-            flexDirection="column"
+            flexDirection={["column", "column", "row", "row"]}
```
Comment (davidli3100) [inline src/components/Footer.js:8]: `{["column", null, "row"]}`

File: src/components/Footer.js [hunk 5]
```diff
@@ -1,44 +1,63 @@
...
-                    sub="Lorem ipsum dolor"
-                    main="Lorem ipsum dolor sit amet"
-                />
-                <Text color="gray" maxWidth="600px" mt="25px" mb="100px">
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                    do eiusmod tempor incididunt ut labore et dolore magna
-                    aliqua.
-                </Text>
+            <Box display="flex" justifyContent="center" alignItems="center">
+                <PseudoBox mx="25px" _hover={{ color: "blue.300" }}>
```
Comment (davidli3100) [inline src/components/Footer.js]: again, check across screensizes (use polypane or something) for responsiveness, and change `mx` with breakpoints where appropriate

File: src/components/Footer.js [hunk 6]
```diff
@@ -1,44 +1,63 @@
...
+                    <a
+                        href="https://www.instagram.com/fraser_hacks/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        <i class="fab fa-instagram fa-2x"></i>
+                    </a>
+                </PseudoBox>
+                <PseudoBox mx="25px" _hover={{ color: "blue.300" }}>
```
Comment (davidli3100) [inline src/components/Footer.js]: see above (responsiveness)

File: src/components/Footer.js [hunk 7]
```diff
@@ -1,44 +1,63 @@
...
+                    <a
+                        href="https://www.facebook.com/fraserhacks/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        <i class="fab fa-facebook fa-2x"></i>
+                    </a>
+                </PseudoBox>
+                <PseudoBox mx="25px" _hover={{ color: "blue.300" }}>
```
Comment (davidli3100) [inline src/components/Footer.js]: see above (responsiveness)

File: src/components/Footer.js [hunk 8]
```diff
@@ -1,44 +1,63 @@
...
+                    <a
+                        href="https://discord.gg/4MDySsa"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        <i class="fab fa-discord fa-2x"></i>
+                    </a>
+                </PseudoBox>
+                <PseudoBox mx="25px" _hover={{ color: "blue.300" }}>
```
Comment (davidli3100) [inline src/components/Footer.js]: see above (responsiveness)

File: src/components/Footer.js [hunk 9]
```diff
@@ -1,44 +1,63 @@
...
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        <i class="fab fa-twitter fa-2x"></i>
+                    </a>
+                </PseudoBox>
             </Box>
-            <Box width="100%">
-                <Img fluid={organizers.childImageSharp.fluid} />
+            <Box mx="25px" mt={["20px", "20px", "0px", "0px"]}>
```
Comment (davidli3100) [inline src/components/Footer.js:138]: see above (responsiveness)

File: src/components/Organizers.js [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+        }
+    `)
+    return (
+        <Box
+            textAlign="center"
+            display="flex"
+            flexDirection="column"
+            alignItems="center"
+        >
+            <Box px="10vw">
```
Comment (davidli3100) [inline src/components/Organizers.js]: see other comments (responsiveness)

File: src/components/Organizers.js [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+                <Title
+                    sub="Lorem ipsum dolor"
+                    main="Lorem ipsum dolor sit amet"
+                />
+                <Text color="gray" maxWidth="600px" mt="25px" mb="100px">
```
Comment (davidli3100) [inline src/components/Organizers.js:31]: see other comments (responsiveness)

File: src/components/SEO.js [hunk 12]
```diff
@@ -26,5 +26,11 @@ export default () => (
             content="FraserHacks 2020 coming soon! Hosted at John Fraser Secondary School"
         />
         <meta property="twitter:image" content="/banner.jpg" />
+        <script
```
Comment (davidli3100) [inline src/components/SEO.js]: just extract the SVGs and load them in as JSX instead of including the entire library

**Review by davidli3100 — CHANGES_REQUESTED** [review]

File: src/components/Hero.js [hunk 13]
```diff
@@ -9,7 +9,7 @@ const Hero = () => {
         query {
             hero: file(relativePath: { eq: "hero.jpg" }) {
                 childImageSharp {
-                    fluid(maxWidth: 2000, maxHeight: 900) {
+                    fluid(maxWidth: 2000, maxHeight: 900, quality: 100) {
```
Comment (err53) [inline src/components/Hero.js:12]: bro why
theres no need, stop killing perf

File: src/components/Slider.js [hunk 14]
```diff
@@ -0,0 +1,171 @@
+import React from "react"
+import { Grid } from "@chakra-ui/core"
+import { graphql, useStaticQuery } from "gatsby"
+import Img from "gatsby-image"
+
+const Slider = () => {
```
[the remark this answers is not in the document]
Comment (err53) [reply src/components/Slider.js:6]: Or maybe pass it more image sizes or smth so loading is better on mobile

File: src/components/Title.js [hunk 15]
```diff
@@ -3,7 +3,7 @@ import { Heading, Box } from "@chakra-ui/core"
 
 export default ({ sub, main }) => (
     <Box textAlign="center">
-        <Heading color="blue" fontSize={["18px", "18px", "24px", "24px"]}>
+        <Heading color="blue.500" fontSize={["18px", "18px", "24px", "24px"]}>
```
Comment (err53) [inline src/components/Title.js]: This shade of blue breaks accessibility.
See: https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1602550331584-57471.report.html#accessibility

File: src/components/Footer.js [hunk 16]
```diff
@@ -1,44 +1,63 @@
...
-                    main="Lorem ipsum dolor sit amet"
-                />
-                <Text color="gray" maxWidth="600px" mt="25px" mb="100px">
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                    do eiusmod tempor incididunt ut labore et dolore magna
-                    aliqua.
-                </Text>
+            <Box display="flex" justifyContent="center" alignItems="center">
+                <PseudoBox mx="25px" _hover={{ color: "blue.300" }}>
+                    <a
```
Comment (err53) [inline src/components/Footer.js:18]: Links do not have a name, accessibility issue.
See: https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1602550331584-57471.report.html#accessibility

File: src/components/Footer.js [hunk 17]
```diff
@@ -1,44 +1,63 @@
...
+                        href="https://www.instagram.com/fraser_hacks/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        <i class="fab fa-instagram fa-2x"></i>
+                    </a>
+                </PseudoBox>
+                <PseudoBox mx="25px" _hover={{ color: "blue.300" }}>
+                    <a
```
Comment (err53) [inline src/components/Footer.js:49]: Accessibility issue, see above.

File: src/components/Footer.js [hunk 18]
```diff
@@ -1,44 +1,63 @@
...
+                        href="https://www.facebook.com/fraserhacks/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        <i class="fab fa-facebook fa-2x"></i>
+                    </a>
+                </PseudoBox>
+                <PseudoBox mx="25px" _hover={{ color: "blue.300" }}>
+                    <a
```
Comment (err53) [inline src/components/Footer.js:80]: Accessibility issue, see above.

File: src/components/Footer.js [hunk 19]
```diff
@@ -1,44 +1,63 @@
...
+                        href="https://discord.gg/4MDySsa"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        <i class="fab fa-discord fa-2x"></i>
+                    </a>
+                </PseudoBox>
+                <PseudoBox mx="25px" _hover={{ color: "blue.300" }}>
+                    <a
```
Comment (err53) [inline src/components/Footer.js:111]: Accessibility issue, see above.

File: src/components/Footer.js [hunk 20]
```diff
@@ -1,44 +1,63 @@
...
+                        rel="noopener noreferrer"
+                    >
+                        <i class="fab fa-twitter fa-2x"></i>
+                    </a>
+                </PseudoBox>
             </Box>
-            <Box width="100%">
-                <Img fluid={organizers.childImageSharp.fluid} />
+            <Box mx="25px" mt={["20px", "20px", "0px", "0px"]}>
+                <a
```
Comment (err53) [inline src/components/Footer.js:139]: Accessibility issue, see above.

**Review by err53 — CHANGES_REQUESTED** [review]
Numerous performance and accessibility regressions introduced with this PR, not suitable for merging.
See: https://googlechrome.github.io/lighthouse-ci/viewer/?baseReport=https%3A%2F%2Fstorage.googleapis.com%2Flighthouse-infrastructure.appspot.com%2Freports%2F1602550019586-10666.report.html&compareReport=https%3A%2F%2Fstorage.googleapis.com%2Flighthouse-infrastructure.appspot.com%2Freports%2F1602550331584-57471.report.html

File: src/components/Hero.js (hunk 13 above)
Comment (RyanL123) [reply src/components/Hero.js:12]: The hero image is NOTICABLY more pixelated without quality 100, so this might have to stay

Comment (err53) [reply src/components/Hero.js:12]: ig its fine for now

File: src/components/Slider.js (hunk 14 above)
[the remark this answers is not in the document]
Comment (err53) [reply src/components/Slider.js:6]: ok sizing is fixed

**Review by err53 — APPROVED** [review]
Lighthouse is still being annoying but its good enough

**Review by davidli3100 — APPROVED** [review]

**Pull request merged**